src/main/java/dev/pgm/events/ready/ReadyManager.java

```diff
@@ -0,0 +1,70 @@
+package dev.pgm.events.ready;
+
+import java.time.Duration;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.party.Party;
+import tc.oc.pgm.start.StartCountdown;
+import tc.oc.pgm.start.StartMatchModule;
+
+public class ReadyManager {
```

Comment:
Would be good to have an isReady method in here as well, then ReadyCommands and other things could just depend on ReadyManager rather than readySystem and readyParties as well

Reply:
On second thoughts, after reading through what you're saying in regards to other stuff it might be nice to make ReadyManager into an interface facade, having an implementation that does all the linking with the logic still in ReadySystem/ReadyParties. Then remove all the dependencies outside of that class on ReadySystem/ReadyParties and have everything instead depend on the interface.

In the future when the ready system is tidied up further/changed a new implementation of the interface can be created cleanly and code that depends on it will not have to be changed